test/specs/components/action.spec.js

```diff
@@ -83,19 +82,23 @@ describe('Action Component', () => {
             className: 'mockedLoading'
         });
 
-        sandbox.stub(userService, 'immediateUpdate');
-        userService.immediateUpdate.resolves();
 
-        sandbox.stub(conversationService, 'postPostback');
-        conversationService.postPostback.resolves();
+        immediateUpdateStub = sandbox.stub().returns(() => Promise.resolve());
```

Comment:
Man, having `.returns(() => Promise.resolve())` everywhere is tedious. I wonder if we could write a sinon extension or something to add a new `returnsResolvingFn` or something like that?